app/pages/Gifts/GiftsPage.presenter/GiftsPage.tsx

```diff
@@ -218,8 +225,12 @@ const GiftsPage: FC = () => {
     setShowModalConsume(false);
   };
 
-  useEffect(getAllGiftCodes, []);
-  useEffect(getFeePmob, []);
+  useEffect(() => {
```

Comment:
useEffect expects a parameter of type `() => void`, which neither of these methods were.